metricbeat/mb/testing/data/data_test.go

```diff
@@ -51,12 +51,41 @@ const (
 var (
 	// Use `go test -generate` to update files.
 	generateFlag = flag.Bool("generate", false, "Write golden files")
+	moduleFlag   = flag.String("module", "", "Choose a module to test")
 )
 
 type Config struct {
-	Type   string
-	URL    string
-	Suffix string
+	// The type of the test to run, usually `http`.
+	Type                      string
```

Comment:
I assume `go fmt` will not be happy with these indentation ;-)

Reply:
Yeah, you were right. `make fmt` executed